eox_tenant/api/v1/tests/test_microsites.py

```diff
@@ -0,0 +1,130 @@
+"""
+Test microsites API v1.
+"""
+
+from django.contrib.auth.models import User
+from mock import patch
+from rest_framework import status
+from rest_framework.test import APIClient, APITestCase
+
+from eox_tenant.models import Microsite
+
+
+class MicrositeAPITest(APITestCase):
+    """
+    Microsite API TestCase.
+    """
+    patch_permissions = patch('eox_tenant.api.v1.permissions.EoxTenantAPIPermission.has_permission', return_value=True)
```

Comment:
could you check if there is a way to do this in the class level, in order to avoid to use the decorator in every case and the useless parameter `_`.  I think we have to test where the user doesn't have the right permissions...

Reply:
we can do something like [this](https://stackoverflow.com/questions/6307761/how-to-decorate-all-functions-of-a-class-without-typing-it-over-and-over-for-eac). but if we want to create a test that does not have the permissions I think it would be a problem.